src/lms-ds-loader/edfi_lms_ds_loader/mssql_lms_operations.py

```diff
@@ -47,6 +47,11 @@ def __callback(session: sa_Session) -> sa_Result:
 
         return 0
 
+    def _query(self, query: str) -> pd.DataFrame:
```

Comment:
This is not being re-used. Unless you _know_ that it will be reused, let's recombine it with `get_processed_files`.